rtspbee-publisher/README.md

```diff
@@ -60,15 +60,31 @@ ffmpeg -re -stream_loop -1 -fflags +igndts -i "./720p_1500kbps_30fps.mp4" \
 rtsp://red5pro.server.com:8554/live/stream1
 ```
 
-## Using rtspbee-publisher.sh
+## RTSP publish bees for single Red5 Pro server
 
+**FILE**: rtspbee-publisher.sh   
+**USAGE**: rtspbee-publisher.sh [endpoint] [app] [streamName] [amount of streams to start] [amount of time to playback] [Red5 Pro server API key] [path to mp4-file]   
+**EXAMPLE**:   
 ``` sh
-./rtspbee-publisher.sh red5pro.server.com:8554 live stream1 10 30 ./720p_1500kbps_30fps.mp4
+./rtspbee-publisher.sh "red5pro.server.com" live stream1 10 10 abc123 /path_to_video_file/bbb_480p.mp4
 ```
 
+## RTSP publish bees for Stream Manager
+
+**FILE**: rtspbee-publisher_sm.sh   
+**USAGE**: rtspbee-publisher_sm.sh [sm_endpoint] [app] [streamName] [amount of streams to start] [amount of time to playback] [Origin API key] [path to mp4-file]   
+**EXAMPLE**:   
+``` sh
+./rtspbee-publisher_sm.sh "https://stream-manager.url/streammanager/api/4.0/event/live/streamname?action=subscribe" live stream1 10 10 abc123 /path_to_video_file/bbb_480p.mp4
```

Comment:
It does not seem logic to put `action=subscribe` for the publish bee.
Is it an error?